src/ru/skypro/Main.java

```diff
@@ -3,6 +3,92 @@
 public class Main {
 
     public static void main(String[] args) {
-	// write your code here
+
+//Task 1
+	byte clientOs = 1;
+    if (clientOs == 0) {
+        System.out.println("Установите версию приложения для iOS по ссылке");
+    }
+    if (clientOs == 1) {
+        System.out.println("Установите версию приложения для Android по ссылке");
+    }
+
+//Task 2
+    int clientDeviceYear = 2014;
+    if (clientOs == 0) {
+        if (clientDeviceYear < 2015){
+            System.out.println("Установите облегченную версию приложения для iOS по ссылке");
+        }else
+            System.out.println("Установите версию приложения для iOS по ссылке");
```

Comment:
скобки лучше всегда явно указывать, чтобы исключить возможность сделать ошибку исключения определенной строки кода из условия. Также, если скобки были в однострочном if, то стоит для согласованности добавить и в else. Аналогично и для других мест 
```suggestion
        } else {
            System.out.println("Установите версию приложения для iOS по ссылке");
        }
```